js/ui/date_box/ui.date_box.mask.parts.js

```diff
@@ -110,6 +110,7 @@ const getLimits = (pattern, date) => {
         H: { min: 0, max: 23 },
         h: { min: 0, max: 23 },
         m: { min: 0, max: 59 },
+        s: { min: 0, max: 59 },
```

Comment:
What about milliseconds in the format? Do we support them?

Reply:
Yep. I'll add it